packages/quantic/force-app/examples/main/lwc/actionAddCustomSortOptions/actionAddCustomSortOptions.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+  <apiVersion>52.0</apiVersion>
```

Comment:
I'm pretty sure that an old api version.

Reply:
copy-pasted another action file, will update this version